src/app/api/sentry-example-api/route.ts

```
@@ -0,0 +1,16 @@
+class SentryExampleAPIError extends Error {
+  constructor(message: string | undefined) {
+    super(message);
+    this.name = 'SentryExampleAPIError';
+  }
+}
+// A faulty API route to test Sentry's error monitoring
+export function GET() {
+  throw new SentryExampleAPIError(
+    'This error is raised on the backend called by the example page.',
+  );
+  return NextResponse.json({ data: 'Testing Sentry Error...' });
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)